.circleci/config.yml

```diff
@@ -1,44 +1,54 @@
-version: 2
-jobs:
-  build:
-    parallelism: 3
+version: 2.1
+
+orbs:
+  ruby: circleci/ruby@0.1.0
+
+executors:
+  ruby_with_postgres:
+    parameters:
+      ruby_tag:
+        description: "The `circleci/postgres` Docker image version tag."
+        type: string
+        default: "latest"
+      postgres_tag:
+        description: "The `circleci/postgres` Docker image version tag."
+        type: string
+        default: "latest"
     docker:
-      - image: circleci/ruby:2.4.2-jessie-node
+      - image: circleci/ruby:<< parameters.ruby_tag >>
         environment:
           BUNDLE_JOBS: 3
           BUNDLE_RETRY: 3
           BUNDLE_PATH: vendor/bundle
           PGHOST: 127.0.0.1
           PGUSER: circleci-demo-ruby
           RAILS_ENV: test
-      - image: circleci/postgres:9.5-alpine
+      - image: circleci/postgres:<< parameters.postgres_tag >>
         environment:
           POSTGRES_USER: circleci-demo-ruby
           POSTGRES_DB: rails_blog
           POSTGRES_PASSWORD: ""
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.2-jessie-node
```

Comment:
Would it be good to use a newer version of Ruby that is also Stretch-based?

Reply:
@lokst Thanks, it sounds great! I updated the ruby version to 2.6.3 and use Stretch-based image.

> https://github.com/CircleCI-Public/circleci-demo-ruby-rails/pull/17/commits/d98f9e04f03e7bc6a411df005dadea0e2317f27e